verto/processors/GenericContainerBlockProcessor.py

```diff
@@ -113,6 +115,7 @@ def run(self, parent, blocks):
             raise ArgumentValueError(self.processor, 'content', content, message)
 
         argument_values['content'] = content
+# think there is something wrong here
```

Comment:
Do you still think something is wrong here? :joy:

Reply:
lol

Reply:
(also, fixed, just not shown as outdated)